cluster/images/obs-csi-plugin/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+HOST_CMD="nsenter --mount=/proc/1/ns/mnt"
```

Comment:
```suggestion
HOST_CMD="/nsenter --mount=/proc/1/ns/mnt"
```
we copied `nsender` to the docker, pls use it.